### PR TITLE
add universal login template

### PR DESCRIPTION
**Pull request by gagalago**
add [universal login template](https://auth0.com/docs/universal-login/new-experience/universal-login-page-templates)

fix https://github.com/go-auth0/auth0/issues/158

### CODE REVIEW

**Comment by gagalago** [comment]
@alexkappa you can review this pull request

File: management/branding.go [hunk 1]
```diff
@@ -116,6 +116,10 @@ type BrandingFont struct {
 	URL *string `json:"url,omitempty"`
 }
 
+type BrandingTemplateUniversalLogin struct {
```
Comment (gagalago) [inline management/branding.go:119]: @alexkappa I am wondering if I need to create a new standalone resource or if I follow the [documentation](https://auth0.com/docs/api/management/v2#!/Branding/get_universal_login) and add it like it is right now?

Comment (gagalago) [reply management/branding.go:119]: I saw that you did already the same in user for example

**Review by relu — APPROVED** [review]
LGTM :+1:

File: management/management.go [hunk 2]
```diff
@@ -257,6 +257,23 @@ func (m *Management) NewRequest(method, uri string, payload interface{}, options
 	return
 }
 
+// NewHtmlRequest returns a new HTTP request.
+func (m *Management) NewHtmlRequest(method, uri string, html string, options ...RequestOption) (r *http.Request, err error) {
```
Comment (alexkappa) [inline management/management.go:261]: We can probably avoid duplicating this method.

**Review by alexkappa — COMMENTED** [review]
Hi @gagalago, thanks for working in this. I've made some changes and merged this in.

**Pull request closed without merging**